holidays/countries/sweden.py

```diff
@@ -126,6 +148,23 @@ def _populate_public_holidays(self):
                 # Sunday.
                 self._add_holiday(tr("Söndag"), dt)
 
+    def _populate_de_facto_holidays(self):
+        """
+        Populate de facto holidays.
+
+        These holidays are treated equivalently to public holidays for working
+        day calculations according to Swedish Annual Leave Law (SFS 1977:480),
+        but are not official public holidays.
+        """
+        # Midsummer Eve (Friday between June 19 and 25).
```

Comment:
```suggestion
        # Midsummer Eve.
```
This and rerun `make l10n` again to auto-address this (or just `make check`)